index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
       </div>
       <div class="col-7_md-12_md-first hero">
         <div>
-          <img class="img md-hidden" src="{{ "/images/hero.svg" | relative_url }}" />
-          <img class="img md-gt-hidden" src="{{ "/images/hero-mobile.svg" | relative_url }}" />
+          <img class="img" srcset="./images/hero.png 1x, ./images/hero@2x.png 2x, ./images/hero@3x.png 3x" alt="Crossplane hero image" />
         </div>
       </div>
     </div>
   </div>
 </div>
 
-<div id="kinds">
+<div id="use-cases">
   <div>
-    <div class="grid-center">
-      <div class="col-10_sm-12">
-        <h2>The many flavors of managed services we support</h2>
+    <div class="grid">
+      <div class="col-6_md-12 img">
+        <img srcset="./images/feature-1.png 1x, ./images/feature-1@2x.png 2x, ./images/feature-1@3x.png 3x" alt="Provision managed services" />
+      </div>
+      <div class="col-6_md-12">
+        <h2>Provision the managed services your applications depend on using ‘kubectl.’</h2>
         <p>
-          Crossplane enables workload portability across clusters and cloud
-          providers for the most common managed services.
+          Securely consume managed services such as PostgreSQL, Redis, Buckets and more, from 
+          your choice of cloud on-premises.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
         </p>
-        <p><img src="{{ "/images/pink.svg" | relative_url }}" /></p>
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/compute.svg" | relative_url }}" /></div>
-        <h4>Compute</h4>
+    </div>
+    <div class="grid">
+      <div class="col-6_md-12">
+        <h2>Extend Crossplane by adding your own Stack.</h2>
+        <p>
+          Add new resources and controllers to Crossplane by extending existing Stacks
+          or building your own.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
+        </p>
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/database.svg" | relative_url }}" /></div>
-        <h4>Databases</h4>
+      <div class="col-6_md-12_md-first img">
+        <img srcset="./images/feature-4.png 1x, ./images/feature-4@2x.png 2x, ./images/feature-4@3x.png 3x"
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper